### ISSUE TITLE
Android: New Note

### ISSUE THREAD

**Issue by qkzoo** [Fmstrat/ownnote#137]
I find crreating new notes very cumbersome in this app.  After I  create a new note, I expect to be able to edit/modify it immediately.  Instead, I have to create a new note, then find it, then open it first.   

Secondly, I should be able to select a group I already created when making a new  note, I'm thinking a dropddown list on the New Note dialog.

Thirdly, after creating a new note in a new group, when I press the groups button on my newly created group, there is a blank page I have to select "Next Page" and then I see my new note.

I look forward to seeing the progress with OwnNote!

**Comment by Fmstrat** [comment]
First off, thanks for all the feedback!

I've spun off two separate tickets targetting v1.1 on your enhancements. I'm going to relegate this issue to just the bug you have reported so I can get it in the next release.

**Comment by Fmstrat** [comment]
Unfortunately I'm not able to reproduce that part. Could you let me know all of the specifics of the new note you're creating that causes this to occur? For instance, when I:

- Tap Create
- Enter "Test" in the name
- Enter "Test" in the group (does not exist)
- Tap Create
- It returns me to the notes listing
- Tap Groups
- Tap Test (the new group)
- I see my note Test, not NEXT PAGE

**Comment by qkzoo** [comment]
You are correct on the steps, and I figured out the problem.  I'm running OwnCloud on a Raspberry Pi, and I'm pretty sure the issue (#3 above) stems from some sort of server lag.  In otherwords, it's not your app, but the server I'm running.   I apologize, I should have done a bit more troubleshooting, etc. before posting the bug.  I plan on purchasing the Pi 2 shortly which should hopefully speed up my OwnCloud install, which is painfully slow!

**Comment by Fmstrat** [comment]
Hmm.. interesting. The requests are sequential, so even if the server took 30 seconds, the issue still shouldn't happen. I'm going to move this out beyond v1.0 to focus on other things, but investigate it further by forcing some delays on my end. Thanks for the feedback.
